src/acvp_kdf135_ikev2.c

```diff
@@ -35,12 +35,56 @@
  * Forward prototypes for local functions
  */
 static ACVP_RESULT acvp_kdf135_ikev2_output_tc (ACVP_CTX *ctx, ACVP_KDF135_IKEV2_TC *stc, JSON_Object *tc_rsp) {
-    json_object_set_string(tc_rsp, "sKeySeed", (const char *)stc->s_key_seed);
-    json_object_set_string(tc_rsp, "sKeySeedReKey", (const char *)stc->s_key_seed_rekey);
-    json_object_set_string(tc_rsp, "derivedKeyingMaterial", (const char *)stc->derived_keying_material);
-    json_object_set_string(tc_rsp, "derivedKeyingMaterialChild", (const char *)stc->derived_keying_material_child);
-    json_object_set_string(tc_rsp, "derivedKeyingMaterialDh", (const char *)stc->derived_keying_material_child_dh);
-    return ACVP_SUCCESS;
+    ACVP_RESULT rv = ACVP_SUCCESS;
+    char *tmp;
+    
+    tmp = calloc(ACVP_KDF135_IKEV2_SKEY_SEED_STR_MAX, sizeof(char));
```

Comment:
Should be +1 for terminator. The STR_MAX is exactly double the maximum allowed number of bytes